src/implementations/vanilla/components/form-radio/form-radio.scss

```diff
@@ -0,0 +1,206 @@
+/**
+ * Radio component
+ * @define radio
+ */
+
+@use "sass:map";
+@use '@ecl/theme-dev/maps/color';
+@use '@ecl/theme-dev/maps/typography';
+@use '@ecl/theme-dev/maps/spacing';
+
+$_label-color: map.get(color.$color, 'grey-100');
+$_label-color-disabled: map.get(color.$color, 'text');
+$_box-size: 1.25rem;
+$_box-border-width: 2px;
+$_box-border-width-checked: 7px;
+$_box-border-color: map.get(color.$color, 'text');
+$_box-border-color-disabled: map.get(color.$color, 'grey-50');
+$_box-border-color-checked: map.get(color.$color, 'grey-100');
```

Comment:
This colour seems to be wrong (same for the next line).
It is blue in current version

Reply:
My bad. Corrected.